src/interfaces/probe.ts

```diff
@@ -1,9 +1,10 @@
 import { RequestConfig } from './request'
 
 export interface Probe {
-  id: number
+  id: string
   name: string
   description?: string
   request: RequestConfig
+  triggerThreshold: number
```

Comment:
What kind trigger threshold is this?
Is it for error case or success case or what?

Reply:
Currently, the threshold is for both success/error case.